### ISSUE TITLE
kubernetes-version key is not applied to helm template

### ISSUE THREAD

**Issue by FatVader** [jtyr/kubeconform-helm#18]
Hi!

There is a problem with kubeconform checking dependent of .Capabilities.KubeVersion templates, like:

```
{{- if and (semverCompare "<1.23-0" (include "kubeVersion" $)) }}
apiVersion: autoscaling/v2beta1
{{- else -}}
apiVersion: autoscaling/v2
{{- end }}
kind: HorizontalPodAutoscaler
...
...
```

If i want to check this chart by 1.19 k8s version it get failed with:

```
> helm kubeconform . --kubernetes-version "1.19.3"                                                                                                                     ○ ERROR: Testing failed: kubeconform failed: failed to run kubeconform: rc=1
stdin - HorizontalPodAutoscaler  failed validation: could not find schema for HorizontalPodAutoscaler
Error: plugin "kubeconform" exited with error
```

As I understood KubeConform is using `helm template` without `--kube-version` param, so the chart above templating by native kubectl version (e.g. 1.24.3) and trying to be checked by 1.19.3 schema.

May be I'm doing something wrong but for me this problem was fixed by adding --kube-version argument to source code of plugin_wrapper.py

```
    if a.kubernetes_version is not None:
        args["helm_tmpl"] += ["--kube-version", a.kubernetes_version]
```

Maybe it will be helpful to you.

**Comment by jtyr** [comment]
Please can you submit a PR with this change?

**Comment by deriamis** [comment]
@jtyr I just ran across this myself, and I created #24 for it.

**Comment by jtyr** [comment]
Fixed via PR #24.
